### DISCUSSION TITLE
Personal Best in "Catch Book"

### DISCUSSION THREAD

**Discussion by Rancher17** [janstehno/cotw-companion#70] [Ideas]
I don't know if this has been suggested before but if not: I would like a way to manually mark my Personal Best score for each species in the Catch Book. After that, I would like it to be marked automatically when I enter a new record that happens to be higher, hopefully with some icon or other visual indication.

**Comment by janstehno** [comment]
Could be implemented. However I think that the filters together with the search bar are strong enough to show you the best trophy of any animal you want. What do you think?

**Reply by Rancher17** [reply]
True, but the idea was to save the user the trouble. However, it is not really that essential either.
